# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2024/10
# model-angelo 1.0.13 has requirement fair-esm==1.0.3, but you have fair-esm 2.0.0.
# model-angelo 1.0.13 has requirement numpy==1.21.*, but you have numpy 1.25.1.
# model-angelo 1.0.13 has requirement pyhmmer==0.7.1, but you have pyhmmer 0.10.6.
diff -ru model-angelo-1.0.13/setup.py model-angelo-1.0.13_requirements/setup.py
--- model-angelo-1.0.13/setup.py	2024-10-24 12:32:09.000000000 +0200
+++ model-angelo-1.0.13_requirements/setup.py	2024-10-25 12:29:24.459511002 +0200
@@ -33,9 +33,9 @@
         "matplotlib",
         "mrcfile",
         "pandas",
-        "fair-esm==1.0.3",
-        "pyhmmer==0.7.1",
+        "fair-esm<=2.0.0",
+        "pyhmmer<=1.10.15",
         "loguru",
-        "numpy==1.21.*",
+        "numpy<=1.25.1",
     ],
 )
